### PR TITLE
SAAS-91 database checks tests

### PR DESCRIPTION
**Pull request by yurkovychv**
automated test cases: 
 PMM-T233
 PMM-T234
 PMM-T236
 PMM-T241
 PMM-T294
 PMM-T295

### CODE REVIEW

**Comment by atymchuk** [comment]
@yurkovychv pls do not merge this before PMM-5937 - that one will require to change some code here

**Comment by yurkovychv** [comment]
@atymchuk sure , no problem😊

File: pmm-app/tests/pages/homePage.js [hunk 1]
```diff
@@ -13,14 +13,19 @@ module.exports = {
       "//span[@class='panel-title-text' and contains(text(), 'Monitored DB Instances')]//../../../..//span[@class='singlestat-panel-value']",
     dashboardHeaderText: 'Percona Monitoring and Management',
     dashboardHeaderLocator: "//div[contains(@class, 'dashboard-header')]",
-    checkUpdateButton: "//button[@class='check-update-button']",
+    checkUpdateButton: "//i[@id='refresh']",
```
Comment (puneet0191) [inline pmm-app/tests/pages/homePage.js:16]: #refresh

Comment (yurkovychv) [reply pmm-app/tests/pages/homePage.js]: done

File: pmm-app/tests/databaseChecks_test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+Feature('Database Failed Checks (disabledSTT state)');
```
Comment (puneet0191) [inline pmm-app/tests/databaseChecks_test.js:1]: Remove DisableSTT State from Feature, since only one now

Comment (yurkovychv) [reply pmm-app/tests/databaseChecks_test.js]: done

**Pull request merged**